provers/sp1/build.rs

```diff
@@ -117,30 +150,87 @@ fn build_program_with_dependencies(
     built_programs.insert(program.to_string());
 }
 
-// fn is_build_enabled() -> bool {
-//     env::var("SKIP_GUEST_BUILD").is_err() && env::var("CARGO_CFG_CLIPPY").is_err()
-// }
-
+/// Returns the output directory for the build artifacts.
 fn get_output_dir() -> PathBuf {
     env::var_os("OUT_DIR")
-        .expect("OUT_DIR environment variable not set")
-        .into()
+        .map(PathBuf::from)
+        .expect("OUT_DIR environment variable is not set. Cannot determine output directory.")
```

Comment:
Can't we gracefully hard-code this to a fallback `OUT_DIR`?

Reply:
`OUT_DIR` is used in [`src/main.rs`](https://github.com/alpenlabs/strata/blob/main/provers/sp1/src/lib.rs). Having a fallback `OUT_DIR` might increase the complexity.